hewr/tests/testthat/test-to_epiweekly_quantile_table.R

```diff
@@ -0,0 +1,82 @@
+create_forecast_data <- function(
+    directory, filename, date_cols, disease_cols, n_draw) {
+  data <- tibble::tibble(
+    date = rep(date_cols, times = length(disease_cols) * n_draw),
+    disease = rep(disease_cols, each = length(date_cols) * n_draw),
+    .value = sample(
+      1:100, length(disease_cols) * length(date_cols) * n_draw,
+      replace = TRUE
+    ),
+    .draw = rep(
+      rep(1:20, each = length(date_cols)),
+      times = length(disease_cols)
+    )
+  )
+  if (length(disease_cols) == 1) {
+    data <- data |>
+      dplyr::rename(!!disease_cols := ".value") |>
+      dplyr::select(-disease)
+  }
+  arrow::write_parquet(data, fs::path(directory, filename))
+}
+
+testthat::test_that("to_epiweekly_quantiles works as expected", {
+  # create temporary directories and forecast files for tests
+  temp_dir <- fs::path_temp("CA")
```

Comment:
Looks to me like the issue in CI is restricted to Windows, and has to do with the cleanup of the temporary files / dirs. I would suggest using`withr::with_tempdir` to handle the temporary directories. That's good practice anyway and I suspect it might be more cross-platform robust. https://withr.r-lib.org/reference/with_tempfile.html

@sbidari